### PR TITLE
Add SignalR counters

### PR DESCRIPTION
**Pull request by BrennanConroy**
Inspired by https://docs.microsoft.com/en-us/aspnet/core/grpc/diagnostics?view=aspnetcore-3.1#metrics

### CODE REVIEW

**Comment by davidfowl** [comment]
Nice! We need more counters 😁

File: aspnetcore/signalr/diagnostics.md [hunk 1]
```diff
@@ -199,6 +199,39 @@ You can attach Diagnostics files to GitHub issues by renaming them so they have
 
 ![Dragging log files on to a GitHub issue](diagnostics/attaching-diagnostics-files.png)
 
+## Metrics
+
+Metrics is a representation of data measures over intervals of time. For example, requests per second. Metrics data allows observation of the state of an app at a high level. .NET gRPC metrics are emitted using <xref:System.Diagnostics.Tracing.EventCounter>.
+
+### SignalR server metrics
+
+SignalR server metrics are reported on the <xref:Microsoft.AspNetCore.Http.Connections> event source.
```
Comment (BrennanConroy) [inline aspnetcore/signalr/diagnostics.md:208]: `<xref:Microsoft.AspNetCore.Http.Connections>`

This is a name not a type reference in the case of event source

File: aspnetcore/signalr/diagnostics.md [hunk 2]
```diff
@@ -199,6 +199,39 @@ You can attach Diagnostics files to GitHub issues by renaming them so they have
...
+
+| Name                    | Description                 |
+|-------------------------|-----------------------------|
+| `connections-started`   | Total connections started   |
```
Comment (BrennanConroy) [inline aspnetcore/signalr/diagnostics.md:212]: The casing was consistent with what the counter produces before

**Comment by scottaddie** [comment]
Thank you for the PR, @BrennanConroy! Publishing this change in a few minutes.

**Pull request merged**